### DISCUSSION TITLE
Vercel Not Working With Google Domain and Email

### DISCUSSION THREAD

**Discussion by tmowreen97** [vercel/community#3739] [Help] [answered]
### Summary

Hey everyone! I'm working on a website redesign for a small business. They have their domain registered with Google, and they also have email registered where people can email to their domain (www.xyz.com, info@xyz.com). I'm hosting the new website on Vercel. Initially I used the custom Nameservers to attach the domain to the new website. That disrupted the email service and the email was no longer working, yet the website was up. Any help? Should I configure with CNAME and A record instead? I received this email from Google Support but unsure if this should be done on Vercel (if so, I'm not sure how) or if it should be done on the Google domain end. I'd appreciate any help!!! This is urgent. Thank you.
<img width="962" alt="Screen Shot 2023-08-23 at 11 35 22 AM" src="https://github.com/vercel/community/assets/96999704/6e43ccda-2109-4f36-b110-9f02924261ba">
<img width="1512" alt="Screen Shot 2023-08-23 at 11 32 41 AM" src="https://github.com/vercel/community/assets/96999704/8d73ec9f-2b91-4f4c-90e9-ac53a7755628">
<img width="1512" alt="Screen Shot 2023-08-23 at 11 32 34 AM" src="https://github.com/vercel/community/assets/96999704/2cbfc592-aa52-4523-be75-097dd8feda85">
<img width="1372" alt="Screen Shot 2023-08-23 at 11 46 33 AM" src="https://github.com/vercel/community/assets/96999704/ba7471e7-c488-4b3f-8f43-cb314ace0465">

### Steps to Reproduce

Please help with best way to attach Google domain WITH email access to Vercel host. Thank you!

**Comment by amyegan** [comment, marked as answer]
If you're still using Vercel nameservers, then you can go to the Domains tab to add `MX` and other DNS records needed for the email service. If you're back to using Google or another registrar's nameservers, then you can manage the email connection there and only add `A` and `CNAME` records to connect the domain to your Vercel project.

**Reply by tmowreen97** [reply]
Thanks so much! It worked with the A and CNAME records.
